data/json/items/tools.json

```diff
@@ -2807,6 +2807,25 @@
     "flags": [ "WATCH" ],
     "magazines": [ [ "battery", [ "light_plus_battery_cell", "light_battery_cell", "light_minus_battery_cell" ] ] ]
   },
+  {
+    "id": "mini_atm",
+    "type": "TOOL",
+    "name": "portable ATM",
+    "description": "Portable POS terminal usually used by merchants to withdraw money from customer's cash cards. Can be used to transfer money to and from the cash cards and access your personal bank account.",
+    "weight": 400,
+    "volume": 2,
+    "price": 50000,
+    "to_hit": -1,
+    "material": "plastic",
+    "symbol": ";",
+    "color": "blue",
+    "ammo": "battery",
+    "initial_charges": 150,
```

Comment:
initial_charges + magazine support in the same tool causes strangeness. Suggest removing this.